cmd/create/runner.go

```diff
@@ -289,66 +287,6 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		return microerror.Mask(err)
 	}
 
-	if r.flag.Wait {
```

Comment:
I don't think we need this in `create` anymore. If we did, we could move the check to a common function for `create --wait` and `wait` commands.